scripts/ci/should-run-short-flow.js

```diff
@@ -26,56 +28,53 @@ const filePatterns = [
  *
  * Short flow should be executed when all changed files match patterns of docs changes.
- *
- * @returns {Boolean}
  */
-module.exports = cwd => {
-	// We target last commit content by default if we're not processing a pull request.
-	let diffTargets = 'HEAD HEAD~1';
+function main() {
+	const options = getOptions( process.argv.slice( 2 ) );
+	const cwd = upath.resolve( options.cwd );
+
 	let changedFilesPaths;
 
-	// TODO: To remove once CKEditor 5 migrates to CircleCI.
-	if ( TRAVIS ) {
-		// Nightly builds should always execute the full flow.
-		if ( TRAVIS_EVENT_TYPE === 'cron' ) {
-			return false;
-		}
+	// For community PRs, always check the entire repository.
+	if ( IS_COMMUNITY_PR ) {
+		return process.exit( 1 );
+	}
 
-		if ( TRAVIS_EVENT_TYPE === 'pull_request' ) {
-			// We have to find merge base in case the feature branch is not up-to-date with target branch.
-			// Without this step, the comparison would include all changes from merging target branch into feature branch.
-			// https://stackoverflow.com/a/25071749
-			const [ commitRangeStart, commitRangeEnd ] = TRAVIS_COMMIT_RANGE.split( '...' );
-			const mergeBase = execSync( `git merge-base ${ commitRangeStart } ${ commitRangeEnd }`, { cwd } ).toString().trim();
+	// Nightly builds should always execute the full flow.
+	if ( CKE5_IS_NIGHTLY_BUILD === '1' || CKE5_IS_NIGHTLY_BUILD === 'true' ) {
+		return process.exit( 1 );
+	}
 
-			diffTargets = `${ mergeBase } ${ commitRangeEnd }`;
-		}
-	} else if ( CIRCLECI ) {
-		// Nightly builds should always execute the full flow.
-		if ( CKE5_IS_NIGHTLY_BUILD === '1' || CKE5_IS_NIGHTLY_BUILD === 'true' ) {
-			return false;
-		}
+	// When processing a build triggered via API, it was triggered by a change in CKEditor 5.
+	// In such a case, run the full flow too.
+	if ( CKE5_IS_EXTERNAL_BUILD === '1' || CKE5_IS_EXTERNAL_BUILD === 'true' ) {
+		return process.exit( 1 );
+	}
 
-		// If processing a pull request build, find all changed files.
-		if ( CIRCLE_PULL_REQUEST ) {
-			const prId = CIRCLE_PULL_REQUEST.split( '/' ).pop();
+	// If processing a pull request build, find all changed files.
+	if ( CIRCLE_PULL_REQUEST ) {
+		const prId = CIRCLE_PULL_REQUEST.split( '/' ).pop();
 
-			changedFilesPaths = execSync( `gh pr view ${ prId } --json files --jq '.files.[].path'`, { cwd } ).toString();
-		}
-	}
+		changedFilesPaths = execSync( `gh pr view ${ prId } --json files --jq '.files.[].path'`, { cwd } ).toString();
+	} else {
+		// We target last commit content by default if we're not processing a pull request.
+		const diffTargets = 'HEAD HEAD~1';
 
-	if ( !changedFilesPaths ) {
 		changedFilesPaths = execSync( `git diff --name-only ${ diffTargets }`, { cwd } ).toString();
 	}
 
-	return doAllFilesMatchPattern(
-		changedFilesPaths.trim().split( '\n' )
+	const files = changedFilesPaths.trim().split( '\n' );
+
+	return process.exit(
+		doAllFilesMatchPattern( files ) ? 0 : 1
```

Comment:
I know that it is not in the scope of the PR, but while reading the code I noticed that it was hard to understand what `filePatterns` meant. What do you think about renaming:
1. `filePatterns` -> `shortFlowFilePatterns`,
2. `doAllFilesMatchPattern` -> `allFilesMatchShortFlow`
Or some other names that have similar meaning.

Reply:
`s/doAllFilesMatchPattern/areAllFilesMatchShortFlow` ?

Reply:
`doAllFilesMatchShortFlow` sound good.

Reply:
Addressed in 2ece7f8b912888ba69d22d964190069fa503d549.